CONTRIBUTING.md

```diff
@@ -237,6 +242,30 @@ body:
 For additional information on using the `dep` tool, see
 "[Performing vendoring for the Kata Containers project](https://github.com/kata-containers/community/blob/master/VENDORING.md)".
 
+### Stable branch backports
+
+Kata Containers maintains a number of stable branch releases. Bug fixes to the
+master branch are selectively applied to (or 'backported') these stable branches.
+
+In order to aid identification of commits that potentially should be backported to
+the stable branches, all PRs submitted must be labelled with one or more of the
+following labels. At least one label that is *not* `stable-candidate` must
+be included.
+
+| Label              | Meaning |
+| -----              | ------- |
+| `bug`              | A bug fix, which will potentially be a backport candidate |
+| `cleanup`          | A cleanup, which will likely not be backported                 |
+| `feature`          | A new feature/enhancement, that will likely not be backported  |
+| `stable-candidate` | A PR selected for backporting - very likely a bug fix          |
+| `vendor`           | A golang vendor update. Might be considered for backport if the vendor update includes critical bug fixes |
+
+In the event of a bug fix PR being selected for backporting to the stable branches,
+the `stable-candidate` label is added if not already present, and the original author
+of the PR is asked if they can submit the relevant backport PRs. For a quick guide
+on how to perform and submit a backport, see the [backport Guide](backport-Guide.md)
```

Comment:
Capitalize this too (I  should have said it before).

Reply:
Doh, good catch! Updated (and sorry if that commit change messes up your review view on github...)